data/saas/dataset/shipstation_dataset.yml

```
@@ -0,0 +1,359 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: shipstation
+    description: A sample dataset representing the Shipstation connector for Fides
+    collections:
+      - name: customer
+        fields:
+          - name: customerId
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: createDate
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: modifyDate
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: name
+            data_categories: [system.operations]
```

You'll need to manually categorize these fields, here's an example for this `name` field
```suggestion
            data_categories: [user.name]
```
Check out this page for the possible data categories https://ethyca.github.io/fideslang/taxonomy/data_categories/
Also take a look at existing SaaS datasets if you want more examples